documentation/modules/deploying/con-deploy-options-scope.adoc

```diff
@@ -31,3 +31,16 @@ Strimzi supports additional deployment options to monitor your deployment.
 * Extract metrics and monitor Kafka components by xref:assembly-metrics-setup-str[deploying Prometheus and Grafana with your Kafka cluster].
 * Extract additional metrics, particularly related to monitoring consumer lag, by xref:proc-metrics-kafka-deploy-options-{context}[deploying Kafka Exporter with your Kafka cluster].
 * Track messages end-to-end by link:{BookURLUsing}#assembly-distributed-tracing-str[setting up distributed tracing^].
+
+== CPU and memory resource limits and requests
+
+By default, the Strimzi deployment does not specify requests and limits for CPU and memory resources.
+It's important for Kafka to have enough of these resources to run properly, but not more than they need if they are sharing resources.
+If you don't set limits, a container could use all the available memory.
```

Comment:
I know IBM says contractions are ok now, but I still think minimizing use of contractions is better for translation purposes.

Reply:
updated